client/app/_user/profile.html

```diff
@@ -11,23 +11,30 @@
         </zf-sub-header-button>
     </zf-sub-header>
     <div class="page page-profile">
-        <div class="profile-header" layout="row">
-            <div class="profile-img">
-                <profile-photo ng-model="$ctrl.currentUser.photoURL" size="48">
-                    <div class="profile-img-content">
-                        <i class="material-icons" ng-click="$ctrl.showUploadImageDialog($event)">add_a_photo</i>
+        <div class="profile-header" layout="row" layout-xs="column">
+            <div layout="row" flex="auto">
+                <div class="profile-img">
+                    <profile-photo ng-model="$ctrl.currentUser.photoURL" size="48">
+                        <div class="profile-img-content">
+                            <i class="material-icons" ng-click="$ctrl.showUploadImageDialog($event)">add_a_photo</i>
+                        </div>
+                    </profile-photo>
+                </div>
+                <div class="profile-wrapper" flex="100">
+                    <div>
```

Comment:
please add a class if you need this `div`

Reply:
fixed